README.md

```diff
@@ -1 +1,84 @@
-Easypay Checkout SDK
+# Easypay Checkout SDK
+
+Easypay's Checkout is intended too facilitate integration with your application.
+
+## Installation:
+
+Install the package with:
+
+```sh
+npm install --save @easypaypt/checkout-sdk
+# or
+yarn add @easypaypt/checkout-sdk
+```
+
+If you do not have `yarn` installed, you can get it with `npm install --global yarn`.
+
+## Usage:
+
+### Import
+
+```js
+import { startCheckout } from "@easypaypt/checkout-sdk"
+
+const checkoutInstance = startCheckout(manifest, { id: "checkout-form" })
+```
+
+| Option     | Type     | Required | Default         | Description                                              |
+| ---------- | -------- | -------- | --------------- | -------------------------------------------------------- |
+| `manifest` | `object` | true     | `undefined`     | server to server response from checkout session creation |
+| `id`       | `string` | false    | `checkout-form` | parameter to link the element in the page                |
+
+### Linking to the Page
+
+```html
+<div id="checkout-form"></div>
+```
+
+### Hidde/Remove checkout from Page
```

Comment:
Hidde -> Hide
Although I wouldn't say hide, I would just say "Remove checkout from page".
(Because hide gives the wrong impression that it's still running and not showing)